.github/workflows/clang.yml

```diff
@@ -0,0 +1,95 @@
+name: Build clang
+# idea - only run on tagged builds?
+# also: opam installs use versions from infer v0.17.0
+# https://github.com/facebook/infer/blob/v0.17.0/opam.locked
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get Changed Files
+        id: files-changed
+        # don't run this if a commit is appropriately tagged, we'll build clang anyway
+        if: ${{ !(startsWith(github.ref, 'refs/tags/v.')) }}
+        uses: trilom/file-changes-action@v1.2.4
+        with:
+          output: ' '
```

Comment:
Why do we need this?

Reply:
This step is the one that checks for changed files from the previous commit. The `if` condition makes it so this step will NOT run if you tag a commit starting with `v.` - I have this workflow set up to always build clang if a commit is already tagged like that. This way you can retroactively tag any commits you'd like GitHub Actions to build for you (e.g. as soon as this is merged) so Infer's build can immediately find releases without you having to push a fake change to the clang/ directory.

Reply:
Sorry I meant the `output: ' '` line. What does it do?

Reply:
Still wondering about this by the way.